app/bundles/LeadBundle/Views/Company/list.html.php

```diff
@@ -153,7 +153,7 @@
                             'mautic_contact_index',
                             [
                                 'search' => $view['translator']->trans('mautic.lead.lead.searchcommand.company').':"'
-                                    .$fields['core']['companyname']['value'].'"',
+                                    .$view->escape($fields['core']['companyname']['value']).'"',
```

Comment:
Are you sure this doesn't conflict with https://github.com/mautic/mautic/pull/7760? That PR is doing the exact opposite 😅

Reply:
I should have been more thorough investigating origins of the change of this line. I and Don expected it was a bad merge conflict resolution. Will fix.